### PR TITLE
Aali/ops 2209 upgrade sandbox mongo role

### PR DESCRIPTION
**Pull request by syed-awais-ali**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?

@edx/devops Review please. Ticket [OPS-2209](https://openedx.atlassian.net/browse/OPS-2209)

### CODE REVIEW

**Comment by feanil** [comment]
@syed-awais-ali can you point me to the sandbox that is running this?

**Comment by syed-awais-ali** [comment]
@feanil, sure. the name of the instance  [syed-awais-ali-sandbox](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Instances:search=syed;sort=desc:instanceId)

File: playbooks/roles/mongo_3_0/tasks/main.yml [hunk 1]
```diff
@@ -270,6 +270,7 @@
   service:
     name: mongod
     state: restarted
+    enabled: true
```
Comment (jibsheet) [inline playbooks/roles/mongo_3_0/tasks/main.yml]: This is the 3.0 role, aren't we installing 3.2?

Comment (feanil) [reply playbooks/roles/mongo_3_0/tasks/main.yml]: Yep, put it in the wrong place. and fixed it in the right place in a later commit.  I'll pull this out before final review.

**Comment by feanil** [comment]
@jibsheet @fredsmith this is ready for final review. 

https://tools-edx-jenkins.edx.org/job/Sandboxes/job/CreateSandbox/4411/console <- from scratch build that built the AMI.
https://tools-edx-jenkins.edx.org/job/Sandboxes/job/CreateSandbox/4414/  <- sandbox from AMI.

**Review by jibsheet — APPROVED** [review]

**Pull request merged**